src/main/app/src/Components/LandscapeComponent/LandscapeOverview/LandscapeOverviewLayout.tsx

```diff
@@ -0,0 +1,165 @@
+            </Button>
+          </Grid>
+          <Grid item xs={12} md={3} lg={2}>
+            <Typography variant='overline' display='block' gutterBottom>
+              Info
+            </Typography>
+            {landscape.description}
+            <br />
+            <br />
+            Identifier: {landscape.identifier}
+            <br />
+            Contact: {landscape.contact || '-'}
+            <br />
+            Teams: {landscape.stats.teams.join(', ')}
+            <br />
+          </Grid>
+
+          <Grid item xs={12} md={3} lg={2}>
+            <Typography variant='overline' display='block' gutterBottom>
+              State
+            </Typography>
+            {landscape.stats.overallState || '-'}
+          </Grid>
+
+          <Grid item xs={12} md={3} lg={2}>
+            <Typography variant='overline' display='block' gutterBottom>
+              Items
+            </Typography>
+            <Typography variant='h2' display='block' gutterBottom>
+              {landscape.stats.items}
+            </Typography>
+            in {landscape.stats.groups} groups
+          </Grid>
+
+          <Grid item xs={12} lg={2}>
+            <Typography variant='overline' display='block' gutterBottom>
+              Last update
+            </Typography>
+            <Typography variant='h3' display='block'>
+              {landscape.stats.lastUpdate?.split(' ')[0] || '-'}
+            </Typography>
+
+            <div>{landscape.stats.lastUpdate?.split(' ')[1] || '-'}</div>
+          </Grid>
+
+          <Grid item xs={12} lg={2}>
+            <Button
+              onClick={() => enterLandscape(landscape)}
+              fullWidth
+              component={Link}
+              className={'button stackedButton'}
+              to={`/landscape/${landscape.identifier}`}
+            >
+              enter
+            </Button>
+
+            <Button
+              onClick={() => enterLog(landscape)}
+              fullWidth
+              className={'button stackedButton'}
+            >
+              log
+            </Button>
+
+            <Button
+              fullWidth
+              className={'button stackedButton'}
+              rel='noopener noreferrer'
+              target={'_blank'}
+              href={
+                process.env.REACT_APP_BACKEND_URL + '/render/' + landscape.identifier + '/map.svg'
+              }
+            >
+              Printable Graph
+            </Button>
+
+            <Button
+              fullWidth
+              className={'button'}
+              rel='noopener noreferrer'
+              target={'_blank'}
+              href={
+                process.env.REACT_APP_BACKEND_URL + '/docs/' + landscape.identifier + '/report.html'
+              }
+            >
+              Printable Report
+            </Button>
+          </Grid>
+        </Grid>
+      );
+    });
+  }
+
+  return (
+    <div className='homeContainer'>
+      <GenericModal modalContent={modalContent} />
```

Comment:
Do we still need this here?

Reply:
yes to display our log component, we could think about using our modal with a context and only import it once at layout, i'll talk with michelle about it soon